src/multicloud-connector/azext_multicloud_connector/aaz/latest/arc_multicloud/solution_configuration/_create.py

```diff
@@ -18,7 +18,10 @@ class Create(AAZCommand):
     """Create a SolutionConfiguration
 
     :example: SolutionConfigurations_CreateOrUpdate
-        az arc-multicloud solution-configuration create --connector-id ymuj --name keebwujt --solution-type nmtqllkyohwtsthxaimsye --solution-settings "{}"
+        az arc-multicloud solution-configuration create --connector-id /subscriptions/{}/resourceGroups/{}/providers/Microsoft.HybridConnectivity/publicCloudConnectors/{} --name mySolutionConfig --solution-type "Microsoft.AssetManagement" --solution-settings periodicSync="true" cloudProviderServiceTypes="ec2,s3" awsGlobalReadOnly="true" cloudProviderRegions="us-east-1,us-east-2" periodicSyncTime="1"
```

Comment:
Bug. solutionconfiguration creation fails when tried with example given in the help message.

![image](https://github.com/user-attachments/assets/14a9a2aa-8f90-421a-8123-ef5fe68bf9ae)